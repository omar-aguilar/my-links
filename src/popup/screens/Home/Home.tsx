import { FunctionComponent } from 'react';
import useDocumentTitle from '../../hooks/useDocumentTitle';

const Home: FunctionComponent = () => {
  useDocumentTitle('My Links - Home');
  return (
    <>
      <div>My Links Home</div>;
    </>
  );
};

export default Home;
