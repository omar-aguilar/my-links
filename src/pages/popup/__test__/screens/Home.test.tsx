import { render, act } from '@testing-library/react';
import Home from '../../screens/Home';

describe('Home', () => {
  it('renders without exploding', async () => {
    let container: HTMLElement = document.createElement('div');
    await act(async () => {
      const component = render(<Home />);
      container = component.container;
    });
    expect(container.ownerDocument.title).toEqual('My Links - Home');
    expect(container.firstChild).toMatchSnapshot();
  });
});
