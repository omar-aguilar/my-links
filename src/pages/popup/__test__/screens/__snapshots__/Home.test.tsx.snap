// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders without exploding 1`] = `
<div>
  <h1>
    My Links Home
  </h1>
  <div
    class="my-2 p-2 rounded shadow-md rounded border border-gray-400"
  >
    <h2
      class="font-bold mb-2"
    >
      Search
    </h2>
    <div
      class="mb-2"
    >
      <label
        class="block"
        for="domain-selector"
      >
        <div
          class="mr-2 text-base text-slate-700 font-bold"
        >
          Domain
        </div>
        <div
          class="flex grow rounded-md shadow-sm ring-1 ring-inset ring-gray-300  focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-600 inline-flex"
        >
          <select
            class="text-gray-500 block focus:outline-none border-none bg-transparent py-1.5 px-1 text-gray-900 placeholder:text-gray-400 focus:ring-0 text-sm"
            id="domain-selector"
          />
        </div>
      </label>
    </div>
    <div
      class="mb-2"
    >
      <label
        class="block"
        for="Short Link"
      >
        <div
          class="mr-2 text-base text-slate-700 font-bold"
        >
          Short Link
        </div>
        <div
          class="flex grow rounded-md shadow-sm ring-1 ring-inset ring-gray-300 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-600"
        >
          <input
            autocomplete="off"
            class="block focus:outline-none flex-1 border-none bg-transparent py-1.5 px-1  text-gray-900 placeholder:text-gray-400 focus:ring-0 text-sm"
            id="Short Link"
            name="Short Link"
            type="text"
            value=""
          />
        </div>
      </label>
    </div>
    <div
      class="mb-2"
    >
      <label
        class="block"
        for="Tag"
      >
        <div
          class="mr-2 text-base text-slate-700 font-bold"
        >
          Tag
        </div>
        <div
          class="flex grow rounded-md shadow-sm ring-1 ring-inset ring-gray-300 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-600"
        >
          <input
            autocomplete="off"
            class="block focus:outline-none flex-1 border-none bg-transparent py-1.5 px-1  text-gray-900 placeholder:text-gray-400 focus:ring-0 text-sm"
            id="Tag"
            name="Tag"
            type="text"
            value=""
          />
        </div>
      </label>
    </div>
  </div>
</div>
`;
